test/SourceKit/Mixed/complete_twice_bridging_header.swift

```diff
@@ -0,0 +1,13 @@
+// REQUIRES: objc_interop
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: %sourcekitd-test -req=complete -pos=8:5 %s -- %s -module-name Mixed -pch-output-dir %t -import-objc-header %S/Inputs/header.h == -req=complete -pos=8:5 %s -- %s -module-name Mixed -pch-output-dir %t -import-objc-header %S/Inputs/header.h | %FileCheck %s --check-prefix=CHECK-MEMBERS
+// RUN: %sourcekitd-test -req=complete -pos=9:1 %s -- %s -module-name Mixed -pch-output-dir %t -import-objc-header %S/Inputs/header.h == -req=complete -pos=9:1 %s -- %s -module-name Mixed -pch-output-dir %t -import-objc-header %S/Inputs/header.h | %FileCheck %s --check-prefix=CHECK-GLOBALS
+// RUN: stat %t/*.pch
```

Comment:
Why `stat` rather than just `ls`? (And, is there a way to test that the PCH is reused instead of recreated?)